### ISSUE TITLE
ElasticDL abstract for OSC talk

### ISSUE THREAD

**Issue by workingloong** [sql-machine-learning/elasticdl#2047]
《ElasticDL - Kubernetes-native 的弹性分布式深度学习框架》

简介：

在深度学习算法开发中，往往存在分布式深度学习程序难写和模型训练效率低的问题：
- 算法研发者往往只了解深度学习，不了解分布式系统，很难写出分布式训练程序；
- 在资源共享的分布式集群上，资源有时充沛有时紧张，资源紧张会导致作业的长时间等待，影响训练效率。

为解决上述问题，ElasticDL 提供了分布式深度学习编程框架和弹性调度能力。使用编程框架，用户只需要填写 forward、loss、feed 三个函数 —— 其中 forward 定义深度学习的前向计算过程；loss 指定训练模型时使用的 loss 函数；feed 用来定制化训练数据到 TensorFlow 的 tensor 的转换过程。ElasticDL 会自动的把用户填写的函数，转换成分布式程序执行。使用弹性调度，用户可以在资源紧张时先使用少量资源运行训练作业；在资源充沛时，训练作业会自动占用被释放出的资源。在分布式策略上，ElasticDL 支持 Parameter Server 和 AllReduce 来支持不同场景的深度学习训练任务。

本 talk 将介绍 ElasticDL 如何高效地开发分布式深度学习程序，并在 Kubernetes 集群上高效地训练深度学习模型

**Comment by QiJune** [comment]
>在分布式深度学习程序开发上，ElasticDL 只需要用户填写 forward、cost、feed 三个函数 —— 其中 forward 定义深度学习的前向计算过程；cost 指定训练模型时使用的 cost 函数；feed 用来定
制化训练数据到 TensorFlow 的 tensor 的转换过程。

--> 这里最后可以点上一句，ElasticDL会自动的把用户填写的函数，转换成分布式程序执行。

>在模型训练上，ElasticDL 通过 Kubernetes API 实现弹性资源调度，在资源紧张时也可以快速拉起分布式训练作业。在分布式策略上，ElasticDL 支持 Parameter Server 和 AllReduce 来支持不同场景的深度学习训练任务。

--> ElasticDL 通过调用 Kubernetes API实现弹性调度，在资源紧张时，可以先使用少量资源运行训练作业；在资源充沛时，可以迅速占用被释放出的资源。

**Comment by brightcoder01** [comment]
cost -> loss

**Comment by workingloong** [comment]
《ElasticDL - Kubernetes-native 的弹性分布式深度学习框架》

简介：

在构建分布式深度学习任务时，通常存在下面问题：
- 分布式系统使用复杂，算法同学不能专注算法开发
- 训练集群资源有限，任务排队时间长
- 大型训练任务可能因为硬件故障中断执行，导致无法按时完成训练

ElasticDL 实现了Kubernetes-native的弹性容错分布式深度学习框架，训练任务可以不重启容忍节点丢失，并可以自动缩减或增加节点数量，减少任务平均等待时间，提升集群资源利用率。用户无需编写额外代码，就可以将Tensorflow, Keras模型使用ElasticDL执行。

本 talk 将结合深度学习 CTR 预估训练任务场景，为大家深度解析 ElasticDL 基于 Kubernetes 实现的弹性分布式深度学习训练的原理；

**Comment by BobLiu20** [comment]
@workingloong 请问一下，这个Talk哪里可以看呢，外部人员能看吗

**Comment by workingloong** [comment]
> 这个

可以的，我们会在 7月2日 视频直播的形式分享 https://www.oschina.net/event/2317294
